docs/en/setup/service-agent/java-agent/README.md

```diff
@@ -159,6 +159,10 @@ property key | Description | Default |
 `plugin.kafka.topic_profilings` | Specify which Kafka topic name for Thread Profiling snapshot to report to. | `skywalking_profilings` |
 `plugin.kafka.topic_management` | Specify which Kafka topic name for the register or heartbeat data of Service Instance to report to. | `skywalking_managements` |
 `plugin.springannotation.classname_match_regex` |  Match spring beans with regular expression for the class name. Multiple expressions could be separated by a comma. This only works when `Spring annotation plugin` has been activated. | `All the spring beans tagged with @Bean,@Service,@Dao, or @Repository.` |
+`plugin.grpclog.service_host` | Specify which grpc server's host for log data to report to. | `127.0.0.1` |
+`plugin.grpclog.service_port` | Specify which grpc server's port for traces data to report to. | `8000` |
+`plugin.grpclog.max_message_size` | Specify the maximum size of log data for grpc client to report to. | `10MB` |
+`plugin.grpclog.upstream_timeout` | How long grpc client will timeout in sending data to upstream. Unit is second.|`30` seconds|
```

Comment:
The config hierarchy seems doesn't make sense. You are putting reporter at level one, but nothing related toolkit. They should use `toolkit.log.grpc.reporter` for all these RPC related things.

Reply:
There is another place of document should be updated

```md
    * If you want to print trace context(e.g. traceId) in your logs, choose the log frameworks, [log4j](Application-toolkit-log4j-1.x.md), 
[log4j2](Application-toolkit-log4j-2.x.md), [logback](Application-toolkit-logback-1.x.md)
```
Should change to 
> If you want to print trace context(e.g. traceId) in your logs, or collect logs, ...

Reply:
Above are still missing. @elk-g